pilot/test/util/kubernetes.go

```diff
@@ -212,7 +212,8 @@ func FetchLogs(cl kubernetes.Interface, name, namespace string, container string
 }
 
 // Eventually retries until f() returns true, or it times out in error
-func Eventually(f func() bool, t *testing.T) {
+func Eventually(name string, f func() bool, t *testing.T) {
+	t.Helper()
```

Comment:
oh nice, does that change the file:line in output to be the caller upon failure ? I didn't know that!